examples/springboot-response-streaming-zip/README.md

```diff
@@ -0,0 +1,92 @@
+# Spring Boot 2 Zip example
+
+A basic file streaming application written with the Spring Boot 2 framework. You can build and test it locally as a typical Spring Boot 2 application.
+
+The application can be deployed in an AWS account using the [Serverless Application Model](https://github.com/awslabs/serverless-application-model). The `template.yaml` file in the root folder contains the application definition.
+
+To run the application we are using the run.sh script located in the resources folder:
+
+```shell
+#!/bin/sh
+
+exec java -cp "./:lib/*" "com.amazonaws.demo.stream.Application"
+```
+
+In the configuration we have to specify the AWS Lambda adapter as a layer and configure the script as handler:
+
+```yaml
+ Properties:
+      MemorySize: 512
+      Handler: run.sh
+      CodeUri: app/
+      Runtime: java11
+      AutoPublishAlias: live
+      SnapStart:
+        ApplyOn: PublishedVersions      
+      Environment:
+        Variables:
+          RUST_LOG: info
+          READINESS_CHECK_PATH: /healthz
+          REMOVE_BASE_PATH: /v1
+          AWS_LAMBDA_EXEC_WRAPPER: /opt/bootstrap
```

Comment:
Please update this template to show the configuration required for response streaming.